api/v1beta1/cryostat_conversion_test.go

```diff
@@ -77,8 +77,11 @@ var _ = Describe("Cryostat", func() {
 })
 
 func tableEntriesTo() []TableEntry {
-	return append(tableEntries(), Entry("WS connections", (*test.TestResources).NewCryostatWithWsConnectionsSpecV1Beta1,
-		(*test.TestResources).NewCryostat))
+	return append(tableEntries(),
+		Entry("WS connections", (*test.TestResources).NewCryostatWithWsConnectionsSpecV1Beta1,
+			(*test.TestResources).NewCryostat),
+		Entry("command config", (*test.TestResources).NewCryostatWithCommandConfigV1Beta1,
+			(*test.TestResources).NewCryostat))
```

Comment:
This test is failing because the conversion leaves an empty NetworkConfigurationList, which is expected. Once serialized to JSON/YAML it will be removed. Of the possibilities to fix this test, the easiest is probably to include the Core and Grafana configs as well. Basically, test that it becomes `NewCryostatWithIngress`.

Reply:
sorry I didn't get the change to test it yet!